tests/queryset_pickle/tests.py

```diff
@@ -240,12 +240,12 @@ def test_pickle_filteredrelation(self):
 
     def test_pickle_filteredrelation_m2m(self):
         group = Group.objects.create(name='group')
-        m2mmodel = M2MModel.objects.create()
+        m2mmodel = M2MModel.objects.create(added=datetime.date(2020, 1, 1))
```

Comment:
Something like `id=1` here would also work just as well. Not sure if you rejected that for some reason.

Reply:
We removed hardcoded PKs from various tests in the past, so I'm trying to avoid them.